doc/sphinx/source/howto/derive-for-training.rst

```diff
@@ -24,27 +25,64 @@ MNIST data.
   - :ref:`update`
 
 
+.. _understanding_ml_ecosystem:
+
+Understanding the ML ecosystem
+===============================
+
+In a :abbr:`Machine Learning (ML)` ecosystem, it makes sense to take advantage 
+of automatable tasks as much as possible. As such, nGraph was designed to use 
+all available "automatic" graph construction endpoints handed down to it from a 
+framework. Our graph-construction API, therefore, needs to operate at a 
+fundamentally lower level than a typical framework's API. And, due to the fact
+that the nGraph :abbr:`Intermediate Representation (IR)` layer is what handles 
+the translation from the framework's endpoints to an optimized (optimizable) ML 
+environment, writing a model directly in nGraph would be somewhat akin to 
```

Comment:
I'd replace everything before the comma with a simple connecting clause (something like "For this reason")

Reply:
Done